examples/example_dotnet/WinSparkleDemoForm.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Reflection;
+using System.Threading;
+using System.Windows.Forms;
+using WinSparkleDotNet;
+
+namespace example_dotnet
+{
+    public partial class WinSparkleDemoForm : Form
+    {
+        private SynchronizationContext _context;
+        private static string _appCastURL = @"http://winsparkle.org/example/appcast.xml";
+
+        public WinSparkleDemoForm()
+        {
+            _context = SynchronizationContext.Current;
+            InitializeComponent();
+            
+            var assembly = Assembly.GetExecutingAssembly();
+            var company = (AssemblyCompanyAttribute)assembly.GetCustomAttributes(typeof(AssemblyCompanyAttribute), false)[0];
+            var name = assembly.GetName().Name;
+            var version = assembly.GetName().Version; //Sparkle can parse basic version scheme like X.X.X.X
+
+            WinSparkle.SetAppcastUrl(_appCastURL);
+            WinSparkle.SetAppDetails(company.Company, name, version.ToString());
```

Comment:
As I know, working with Assembly info. with C# is a pain. I will do some research but I promises nothing.
For the appcast URL, we can add it to AssemblyInfo but it will need to add a class just for this. Not very "simple" :/